clients/api-entreprise/association/index.ts

```diff
@@ -183,12 +185,26 @@ export interface Meta {
 }
 
 /**
- * GET
+ * GET documents from API Entreprise
  */
-export const clientApiEntrepriseAssociation = async () => {
-  const response = mock as IApiEntrepriseAssociation;
+export const clientApiEntrepriseAssociation = async (siren: Siren) => {
+  if (!process.env.API_ENTREPRISE_URL || !process.env.API_ENTREPRISE_TOKEN) {
+    throw new HttpUnauthorizedError('Missing API Entreprise credentials');
+  }
 
-  return mapToDomainObject(response.data);
+  const response = await httpGet(
```

Comment:
Je ne sais pas comment faire ça exactement ici, mais il y a du Rate-Limiting sur API Entreprise à potentiellement gérer: https://entreprise.api.gouv.fr/developpeurs#respecter-la-volum%C3%A9trie

Je vois qu'il existe des libs pour ça (which is nice): https://github.com/aishek/axios-rate-limit not sure que ça soit bien maintenu.